govcd/openapi_org_network_test.go

```diff
@@ -367,3 +370,31 @@ func runOpenApiOrgVdcNetworkTest(check *C, vdc *Vdc, orgVdcNetworkConfig *types.
 	_, err = vdc.GetOpenApiOrgVdcNetworkById(orgVdcNet.OpenApiOrgVdcNetwork.ID)
 	check.Assert(ContainsNotFound(err), Equals, true)
 }
+
+type dhcpConfigFunc func(check *C, vdc *Vdc, orgNetId string)
+
+func nsxtRoutedDhcpConfig(check *C, vdc *Vdc, orgNetId string) {
+	dhcpDefinition := &types.OpenApiOrgVdcNetworkDhcp{
+		Enabled: takeBoolPointer(true),
+		DhcpPools: []types.OpenApiOrgVdcNetworkDhcpPools{
+			{
+				Enabled: takeBoolPointer(true),
+				IPRange: types.OpenApiOrgVdcNetworkDhcpIpRange{
+					StartAddress: "2.1.1.200",
+					EndAddress:   "2.1.1.201",
+				},
+			},
+		},
+	}
+	updatedDhcp, err := vdc.UpdateOpenApiOrgVdcNetworkDhcp(orgNetId, dhcpDefinition)
+	check.Assert(err, IsNil)
+
+	check.Assert(dhcpDefinition, DeepEquals, updatedDhcp.OpenApiOrgVdcNetworkDhcp)
+
+	// VCD Versions before 10.2 do not allow to perform "DELETE" on DHCP pool
```

Comment:
Just thought to leave a note. It does not matter even if API version used is V34.0 (the maximum for 10.1)